docs/en_US/install/juxt.md

```diff
@@ -17,7 +17,10 @@ Nimble and Inkay already take care of in game patches for miiverse, so you're al
 
 <br>
 
-### Applet Patch
+### Applet Patch (Tiramisu)
+**Aroma users**: The Aroma patcher already handles the applet patch and redirects the Miiverse applet to Juxtaposition.
+The below instructions are **only for Tiramisu users.**
+
```

Comment:
You mentioned wanting an infobox, I assume you meant for here. We do have an implementation of an infobox already, seen here https://github.com/PretendoNetwork/website/blob/master/docs/en_US/install/3ds.md?plain=1#L3-L5

Reply:
Is there a warning infobox too? Or just `tip` for now

Reply:
Just tip right now, but a warning can easily be added. The emoji can be changed to the warning emoji for now too

This is what the infobox looks like when rendered right now ![Screenshot from 2023-04-11 07-54-33](https://user-images.githubusercontent.com/27011796/231155290-e92d218b-5e0b-467c-bbf8-00c120d12eb8.png)

Reply:
I'll just change the emoji I think